tests/integration/dynamic-controllers-express-server.spec.ts

```diff
@@ -527,6 +550,28 @@ describe('Express Server', () => {
           expect(body.arrayUniqueItem).to.deep.equal(bodyModel.arrayUniqueItem);
           expect(body.model).to.deep.equal(bodyModel.model);
           expect(body.mixedUnion).to.deep.equal(bodyModel.mixedUnion);
+
+          expect(body.nestedObject.floatValue).to.equal(bodyModel.nestedObject.floatValue);
+          expect(body.nestedObject.doubleValue).to.equal(bodyModel.nestedObject.doubleValue);
+          expect(body.nestedObject.intValue).to.equal(bodyModel.nestedObject.intValue);
+          expect(body.nestedObject.longValue).to.equal(bodyModel.nestedObject.longValue);
+          expect(body.nestedObject.booleanValue).to.equal(bodyModel.nestedObject.booleanValue);
+          expect(body.nestedObject.arrayValue).to.deep.equal(bodyModel.nestedObject.arrayValue);
+
+          expect(new Date(body.nestedObject.dateValue)).to.deep.equal(new Date(bodyModel.nestedObject.dateValue));
+          expect(new Date(body.nestedObject.datetimeValue)).to.deep.equal(new Date(bodyModel.nestedObject.datetimeValue));
+
+          expect(body.nestedObject.numberMax10).to.equal(bodyModel.nestedObject.numberMax10);
+          expect(body.nestedObject.numberMin5).to.equal(bodyModel.nestedObject.numberMin5);
+          expect(body.nestedObject.stringMax10Lenght).to.equal(bodyModel.nestedObject.stringMax10Lenght);
+          expect(body.nestedObject.stringMin5Lenght).to.equal(bodyModel.nestedObject.stringMin5Lenght);
+          expect(body.nestedObject.stringPatternAZaz).to.equal(bodyModel.nestedObject.stringPatternAZaz);
+
+          expect(body.nestedObject.arrayMax5Item).to.deep.equal(bodyModel.nestedObject.arrayMax5Item);
+          expect(body.nestedObject.arrayMin2Item).to.deep.equal(bodyModel.nestedObject.arrayMin2Item);
+          expect(body.nestedObject.arrayUniqueItem).to.deep.equal(bodyModel.nestedObject.arrayUniqueItem);
+          expect(body.nestedObject.model).to.deep.equal(bodyModel.nestedObject.model);
+          expect(body.nestedObject.mixedUnion).to.deep.equal(bodyModel.nestedObject.mixedUnion);
```

Comment:
We might as well have a positive test for intersection here. I think that’s the only coverage that I couldn’t find. Great work on this PR! :)

Reply:
https://github.com/lukeautry/tsoa/pull/415/commits/d0fe540cc6734d10a10b189fe3c24c4722979505